.travis.yml

```diff
@@ -1,19 +1,20 @@
 sudo: false  # run travis jobs in containers
 language: python
+python:
+  - "2.6"
```

Comment:
@berkerpeksag I don't think they're needed on the final PyPy entry, but they are on these. Without them, YAML parses the Python version number as a decimal instead of a string.